### PR TITLE
Clarify SMs/CUs/EUs belonging to nVidia/AMD/Intel.

### PR DESCRIPTION
**Pull request by garloff**
Also add a line clarifying the order of extensions being fixed.

### CODE REVIEW

File: Standards/scs-0100-v3-flavor-naming.md [hunk 1]
```diff
@@ -428,16 +431,16 @@ This extension provides more details on the specific GPU:
 - pass-through (`G`) vs. virtual GPU (`g`)
 - vendor (X)
 - generation (N)
-- no. of compute units / SMs / EUs (M) that are exposed (for pass-through) or assigned
+- no. (M) of compute units (CUs) / streaming multiprocessors (SMs) / execution units (EUs) - that are exposed (for pass-through) or assigned
```
Comment (mbuechse) [inline Standards/scs-0100-v3-flavor-naming.md:434]: I think I would prefer to use a generic term (i.e., one that is not vendor-specific). Then we can mention that the vendors use other terms for the "same" thing (the thing that we want to distinguish, however "same").

**Review by mbuechse — APPROVED** [review]
I approve, but I think it could be improved by using a term that is not vendor specific.

File: Standards/scs-0100-v3-flavor-naming.md (hunk 1 above)
Comment (anjastrunk) [reply Standards/scs-0100-v3-flavor-naming.md:434]: I agree, I will ask our GPU expert for such a name.

Comment (anjastrunk) [reply Standards/scs-0100-v3-flavor-naming.md:434]: Let's choose "processing units".

```suggestion
- no. (M) of processing units that are exposed (for pass-through) or assigned. A processing unit performs computational work. Vendors may have specific names for processing units, such as execution units (EUs) by Intel or "streaming multiprocessors (SMs)" by NVIDIA calls.
```

**Review by anjastrunk — APPROVED** [review]

File: Standards/scs-0100-v3-flavor-naming.md (hunk 1 above)
Comment (garloff) [reply Standards/scs-0100-v3-flavor-naming.md:434]: As there are three major GPU vendors, shouldn't we have examples for all three in the text to treat them equally?

Comment (anjastrunk) [reply Standards/scs-0100-v3-flavor-naming.md]: I agree and will figure out, how AMD calls processing units.

Comment (mbuechse) [reply Standards/scs-0100-v3-flavor-naming.md]: Anja, I have already incorporated this point. This whole PR has been merged. It's done!

**Pull request merged**